src/evo/providertx.cpp

```diff
@@ -179,8 +183,14 @@ bool CheckNewAssetTx(const CTransaction &tx, const CBlockIndex *pindexPrev, CVal
     }
 
     int hashlen = assetTx.referenceHash.length();
-    if ((hashlen != 0 && hashlen != 46) || (hashlen == 46 && assetTx.referenceHash.substr(0, 2) != "Qm")) {           
+    //ipfs alway start with 'Qm' and length 46
+    if ((hashlen == 46 && assetTx.referenceHash.substr(0, 2) != "Qm")) {
```

Comment:
i think we discuss to support more then just ipfs hash so start wtih qm should not be required

Reply:
fixed